eva/catalog/models/df_metadata.py

```diff
@@ -25,19 +25,19 @@ class DataFrameMetadata(BaseModel):
     _name = Column("name", String(100), unique=True)
     _file_url = Column("file_url", String(100))
     _unique_identifier_column = Column("identifier_column", String(100))
-    _is_video = Column("is_video", Boolean)
+    _table_type = Column("table_type", Integer)
```

Comment:
Shall we use the enum type? https://stackoverflow.com/questions/2676133/best-way-to-do-enum-in-sqlalchemy